### PR TITLE
Making some translations/corrections for pt-BR

### PR DESCRIPTION
**Pull request by Garccez**
I've made some translations/corrections for pt-BR that consist mainly in:
- Changing 'kit' to 'pacote' (per official use in the website and use in Brazilian news);
- Some minor grammatical corrections;
- Correction of minor consistency errors;
- Translation of a few (very few) strings.

### CODE REVIEW

File: src/locale/locales/pt-BR/messages.po [hunk 1]
```diff
@@ -681,7 +681,7 @@ msgstr "Senha de Aplicativo excluída"
 
 #: src/view/com/modals/AddAppPasswords.tsx:138
 msgid "App Password names can only contain letters, numbers, spaces, dashes, and underscores."
-msgstr "O nome da Senha de Aplicativo só pode conter letras, números, traços e sublinhados."
```
Comment (iagocavalcante) [inline src/locale/locales/pt-BR/messages.po:684]: This message its good enough

File: src/locale/locales/pt-BR/messages.po [hunk 2]
```diff
@@ -1021,7 +1021,7 @@ msgstr "Câmera"
 
 #: src/view/com/modals/AddAppPasswords.tsx:180
 msgid "Can only contain letters, numbers, spaces, dashes, and underscores. Must be at least 4 characters long, but no more than 32 characters long."
-msgstr "Só pode conter letras, números, espaços, traços e sublinhados. Deve ter pelo menos 4 caracteres, mas não mais de 32 caracteres."
```
Comment (iagocavalcante) [inline src/locale/locales/pt-BR/messages.po:1024]: Same here, previous translation its correct

**Review by iagocavalcante — COMMENTED** [review]
Suggestions to keep an old translation

File: src/locale/locales/pt-BR/messages.po (hunk 1 above)
Comment (Garccez) [reply src/locale/locales/pt-BR/messages.po:684]: Sure ;)

File: src/locale/locales/pt-BR/messages.po [hunk 3]
```diff
@@ -2208,7 +2208,7 @@ msgstr "Editar Perfil"
 
 #: src/screens/StarterPack/StarterPackScreen.tsx:554
 msgid "Edit starter pack"
-msgstr "Editar kit incial"
+msgstr "Editar pacote incial"
```
Comment (gleydson) [inline src/locale/locales/pt-BR/messages.po:2211]:
```suggestion
msgstr "Editar pacote inicial"
```

Comment (gleydson) [reply src/locale/locales/pt-BR/messages.po:2211]: typo

**Review by gleydson — COMMENTED** [review]
LGTM

**Comment by lsuperti** [comment]
I think "hífen" is a better word for "dash". To me at least, "traço" isn't as immediately clear as "hífen" when referring to dashes.

**Comment by haileyok** [comment]
Lovely! Thanks @Garccez, @iagocavalcante, and @gleydson! I'll get this in later at some point today. We generally leave these PRs up for a bit so lots of discussion can take place in them, but wanted to push out the iterations a little quicker for all of you coming over en masse. Appreciate all the help! 💙

**Comment by lsuperti** [comment]
Since we are looking into the translation for "dashes", isn't "hífens" better? It isn't as truthful to the original text, but "traços" looks a little too generic and not as clear to me.

**Comment by haileyok** [comment]
Thanks @lsuperti. Will hold to merge this until you get feedback on those suggestions.

**Comment by Garccez** [comment]
@lsuperti I actually considered putting hífens instead of riscas/traços, but I don't know what are the others' opinions on it. I'm very supportive on changing it to hífens, though ;)

**Comment by fabiohcnobre** [comment]
I was working on a new fix. #5099 I have already put this review of this PR in PR #5099

**Comment by haileyok** [comment]
@Garccez @fabiohcnobre Do we need to merge both this and https://github.com/bluesky-social/social-app/pull/5099 or just one?

**Comment by fabiohcnobre** [comment]
@haileyok 
Everything that was done here I did in #5099

**Comment by haileyok** [comment]
merged in through https://github.com/bluesky-social/social-app/pull/5099

**Pull request closed without merging**